### PR TITLE
Vscode

### PR DESCRIPTION
**Pull request by wallento**
support for vscode development:

 - dev container setup
 - debug target (chrome launch not fully working in remote target, but debug does!)

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      }
+    }
+  ],
+  "compounds": [
+    {
+      "name": "fullstack: nuxt",
+      "configurations": ["server: nuxt", "client: chrome"]
+    }
+  ]
+}
```
Comment (imphil) [inline .vscode/launch.json:33]: Missing trailing newline.

File: .devcontainer/devcontainer.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+	// Use 'postCreateCommand' to run commands after the container is created.
+	"postCreateCommand": "yarn install",
+
+	// Configure tool-specific properties.
+	"customizations": {
+		"vscode": {
+			"extensions": [
+				"vue.volar",
+				"cpreston321.nuxt-vscode"
```
Comment (imphil) [inline .devcontainer/devcontainer.json:23]: I would add 
* Vue.vscode-typescript-vue-plugin
* Nuxt.mdc (for auto-complete of Nuxt Content-components within Markdown files)
* bradlc.vscode-tailwindcss (extremely helpful for CSS class auto-complete)

**Review by imphil — APPROVED** [review]
LGTM, with two small comments.

(I would still use the dev environment without additional container, since it's just two packages, but it's good to have the choice!)

**Comment by wallento** [comment]
Yeah, it is intended for the casual user, but the devcontainer resolves the complaint of not being able to work with it

**Pull request merged**